mathosphere-core/src/log4j.properties

```diff
@@ -1 +1 @@
-log4j.rootLogger=OFF
\ No newline at end of file
+log4j.rootLogger=off
```

Comment:
in general, try to avoid changes that have no effect. Otherwise git blame might be misleading